README.md

```diff
@@ -1,4 +1,44 @@
 cerberus
 ========
 
-Automatic [alert system](http://vitillo.github.io/cerberus/dashboard/) for telemetry histograms.
+Automatic [alert system](http://mozilla.github.io/cerberus/dashboard/) for [telemetry](http://telemetry.mozilla.org/) histograms.
+
+Cerberus detects changes in the distribution of histograms (histogram regressions), and posts alerts for these to the [medusa](https://github.com/mozilla/medusa) alert service.
+
+Cerberus is also responsible for sending out email reminders for histograms that are expiring in upcoming versions.
+
+The production instance is currently hosted on [telemetry-regression-alerts](ec2-52-13-120-199.us-west-2.compute.amazonaws.com) on the us-east-1 AWS region. All the production files live in `/mnt/telemetry/cerberus/`. There is a cronjob 
+
+Code Overview
+-------------
+
+`run.sh` is a shell script that will set up all the dependencies, then do a full run of the Cerberus detectors. These are the components used in the script:
+
+* `exporter/export.js` downloads the histogram evolutions from the [v4 aggregates API](https://github.com/vitillo/python_mozaggregator).
+  * This script uses a custom fork of [telemetry-js-node](https://github.com/Uberi/telemetry-js-node/tree/v4-pipeline) that uses the v4 pipeline.
```

Comment:
`telemetry-js-node` uses Telemetry.js v1, which is on the v2 pipeline; this fork is a version that uses Telemetry.js v2, in order to support the v4 pipeline. The library needed [significant changes](https://github.com/Uberi/telemetry-js-node/tree/v4-pipeline) in order to account for the differences.

From what I can see, there are three options for this fork:

1. Add a flag to telemetry-js-node that selects which version to load.
  * Would be a little strange since setting a flag would cause a different, incompatible API to be exposed.
2. Publish the fork as a separate module, under a different name.
  * This fork doesn't share that much code with the original, and it's a pretty small library to begin with.
  * The dependency would then be `telemetry-js-v2-node` or something like that.
3. Keep it as a fork.
  * Confusing to other people who want to use it.

Which one would be better?